python/vowpalwabbit/pyvw.py

```diff
@@ -8,6 +8,22 @@
 class SearchTask():
     """Search task class"""
     def __init__(self, vw, sch, num_actions):
+        """
+        Parameters
+        ----------
+
+        vw : vw object
```

Comment:
Is this able to link to the docs for the vw object?

Reply:
This doesn't link to the vw object. We can use "See Also" sub-section of  sphinx docs which is used to recommend the user to have a look at some particular doc related to this.